### PR TITLE
Update files-to-onedrive.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that in case of a shared computer the storage could get consumed quickly

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10423

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/files-to-onedrive.md [hunk 1]
```diff
@@ -48,4 +48,7 @@ After you sign up for Microsoft 365 Business Premium, you&#39;ll want to copy yo
 In addition to working on your OneDrive files locally, you can access them from your web browser:
 
 1. In your web browser, sign in to office.com with your work email.
-2. Choose  **OneDrive**. All of your files are listed, and you can work on them in the browser from any computer.
\ No newline at end of file
+2. Choose  **OneDrive**. All of your files are listed, and you can work on them in the browser from any computer.
+
+> [!NOTE]  
+> When multiple users use shared computer and sync files via onedrive locally, the storage could get consumed very quickly.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/files-to-onedrive.md]:
```suggestion
> When multiple users use shared computer and sync files via onedrive locally, the storage could get consumed very quickly.
```
```suggestion
> When multiple users use a shared computer and sync files via OneDrive locally, the available storage might  be consumed very quickly.
```

Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/files-to-onedrive.md]:
```suggestion
> When multiple users use a shared computer and sync files via OneDrive locally, the available storage might be consumed very quickly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/setup/files-to-onedrive.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/admin/setup/files-to-onedrive.md]: It says it is outdated

Comment (msbemba) [reply microsoft-365/admin/setup/files-to-onedrive.md]: @JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @msbemba, it seems fine.

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**